sdk/search/Azure.ResourceManager.Search/src/Azure.ResourceManager.Search.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.1.0-beta.1</Version>
+    <Version>1.1.0</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.1</ApiCompatVersion>
+    <ApiCompatVersion>1.0</ApiCompatVersion>
```

Comment:
Please revert this back, we should never touch the ApiCompatVersion portion as the comment above states